scripts/install_xcpng.py

```diff
@@ -237,7 +198,7 @@ def main():
         hdd = 'nvme0n1' if vm.is_uefi else 'sda'
         generate_answerfile(tmp_local_path, installer, args.host, args.target_hostname, args.action, hdd,
                             netinstall_gpg_check)
-        generate_boot_conf(tmp_local_path, installer, args.action)
+        pxe.generate_boot_conf(tmp_local_path, installer)
```

Comment:
I don't understand this change. This option was used to have 3 actions: install, upgrade or restore.
I don't see how is it managed in the pxe file. It doesn't seems to exist anymore.

Reply:
`generate_boot_conf` only uses the action to set `rt=1` as a workaround in the `restore` case, I want to have a look at that to see if we can do better (e.g. an answerfile solution)

Reply:
this was reverted and split into #252